cmake/projects/Boost/schemes/url_sha1_boost.cmake.in

```diff
@@ -154,6 +154,7 @@ ExternalProject_Add(
     install
     ${boost_list}
     "--prefix=@HUNTER_PACKAGE_INSTALL_PREFIX@"
+    --ignore-site-config
```

Comment:
Since build instructions changed, please update `PACKAGE_INTERNAL_DEPS_ID` of all `hunter.cmake` files on Boost package:
* https://docs.hunter.sh/en/latest/reference/user-modules/hunter_download.html